### DISCUSSION TITLE
Jlink's Ozone alternative for GDB

### DISCUSSION THREAD

**Discussion by mrx23dot** [blackmagic-debug/blackmagic#1703] [General]
Is there anything close to the Ozone from Segger as a GUI for GDB?

![image](https://github.com/blackmagic-debug/blackmagic/assets/4163396/145b6663-c5c4-44c5-a475-261e1afa8034)

https://www.segger.com/products/development-tools/ozone-j-link-debugger/#gallery-6

It's a well designed **debugging only** interface, running on all native and purposefully built for debugging.

Loads .elf, burns it in, debugs it, without IDE.

For GDB I have only seen debuggers as addons on top of Eclipse/VScode.

**Comment by dragonmux** [comment]
We have no idea, and it's rather outside of the scope of this project - you'd need to ask the GDB folks. We only plug into GDB via its remote debugging protocol and have no control over such matters.

Perhaps you might want to investigate VSCode as its Cortex Debug extension integrates BMP into its debugger infrastructure, or something like GEF for command-line GDB?
